creator/management/commands/c2m2_load.py

```diff
@@ -0,0 +1,358 @@
+                    "Namespace for " + ROOT_PROJECT_NAME,
+                    "Namespace for " + ROOT_PROJECT_NAME,
+                ]
+            )
+
+    def write_project(self):
+        """
+        Create projects, one for Kids first and one for each study.
+        """
+        logger.info("Writing project info")
+        with open(
+            os.path.join(self.out_dir, "project.tsv"),
+            "w",
+            newline="\n",
+        ) as f:
+            writer = csv.writer(f, delimiter="\t")
+            # Header
+            writer.writerow(
+                [
+                    "id_namespace",
+                    "local_id",
+                    "persistent_id",
+                    "creation_time",
+                    "abbreviation",
+                    "name",
+                    "description",
+                ]
+            )
+            # The Kids First project itself
+            writer.writerow(
+                [
+                    ROOT_PROJECT_NS,
+                    ROOT_PROJECT_LOCAL_ID,
+                    None,
+                    None,
+                    ROOT_PROJECT_ABBR,
+                    ROOT_PROJECT_NAME,
+                    ROOT_PROJECT_DESCRIPTION,
+                ]
+            )
+            # Write a project for each study
+            # logger.info(f"write study to projects {self.studies}")
+            for kf_id, study in self.studies.items():
+                print(study)
+                writer.writerow(
+                    [
+                        ROOT_PROJECT_NS,
+                        kf_id,
+                        None,
+                        None,
+                        kf_id,
+                        study["short_name"],
+                        study["name"],
+                    ]
+                )
+
+    def write_project_in_project(self):
+        """
+        Create subprojects, one for each Kids First study.
+        """
+        logger.info("Writing subproject info")
+        with open(
+            os.path.join(self.out_dir, "project_in_project.tsv"),
+            "w",
+            newline="\n",
+        ) as f:
+            writer = csv.writer(f, delimiter="\t")
+            # Header
+            writer.writerow(
+                [
+                    "parent_project_id_namespace",
+                    "parent_project_local_id",
+                    "child_project_id_namespace",
+                    "child_project_local_id",
+                ]
+            )
+            # Content
+            for study in self.studies:
+                writer.writerow(
+                    [
+                        ROOT_PROJECT_NS,
+                        ROOT_PROJECT_LOCAL_ID,
+                        ROOT_PROJECT_NS,
+                        study,
+                    ]
+                )
+
+    def process_study_files(self, study):
+        files = self.gather_study(study)
+        self.write_study_files(study, files)
+
+    def gather_study(self, study):
```

Comment:
You could use kf-utils-python for this. It's going to become a dependency once we merge the `ingest-app` branch anyway. https://github.com/kids-first/kf-utils-python#dataservice